### PR TITLE
Fix #158: Explicit dev deps

### LINKED ISSUE #158: Make devDependencies explicit
Currently both our `dependencies` and `devDependencies` are dumped in `dependencies` in our `package.json`. This is because by default, Travis runs `yarn install` in production mode, and so ignores `devDependencies`.

To enable us to make our `devDependencies` explicit, we need to pass [`production=false`](https://github.com/yarnpkg/yarn/pull/2223) to Travis. This will ensure both `dependencies` and `devDependencies` are installed during CI.

### PR DESCRIPTION
**Pull request by Jameskmonger**
## What does this change?

Fixes #158 

## What is the benefit?

Dev dependencies are explicitly stated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You will need:
 Install dependencies:
 
 ```
-$ yarn install
+$ yarn install --production=false
```
Comment (SiAdcock) [inline README.md]: It isn't necessary to specify this flag for development. `devDependencies` will always be installed.

Comment (Jameskmonger) [reply README.md]: Oh? What do you mean in #158?

> we need to pass `production=false` to Travis

Comment (SiAdcock) [reply README.md]: Yeah, it's necessary to pass this to Travis. Travis has `NODE_ENV` set to `production` when it runs `yarn install`, so `devDependencies` aren't installed (unless you pass the `--production=false` flag.

However developers (who this README is aimed at) will not have `NODE_ENV` set to `production` on their dev machines, so it is not necessary for them to set the `--production=false` flag to ensure `devDependencies` will be installed.

Does that make sense?

Comment (Jameskmonger) [reply README.md]: Oops I've just noticed that this comment thread is on the readme, not on travis.yml! I'll fix this 😆 😨

Comment (SiAdcock) [reply README.md]: Thanks man. Otherwise looks perfect 🏅 💎

**Comment by Jameskmonger** [comment]
Fixed @SiAdcock

**Review by SiAdcock — APPROVED** [review]
Beautiful stuff, thanks man! Merge at will 👍

**Pull request merged**